Casks/p/piezo.rb

```diff
@@ -1,19 +1,40 @@
 cask "piezo" do
-  version "1.8.2"
   sha256 :no_check
 
-  url "https://rogueamoeba.com/piezo/download/Piezo.zip"
+  on_ventura :or_older do
+    version "1.8.2"
+
+    url "https://rogueamoeba.com/piezo/download/Piezo.zip"
+
+    # NOTE: The `system` value will need to be kept up to date with the latest
+    # macOS Ventura version (e.g. 1366 for 13.6.6).
+    livecheck do
+      url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=1366&bundleid=com.rogueamoeba.Piezo&platform=osx&version=#{version.no_dots}8000"
```

Comment:
Thoughts on this to make updates a bit easier?

```suggestion
      system_version = "1366"
      url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=#{system_version}&bundleid=com.rogueamoeba.Piezo&platform=osx&version=#{version.no_dots}8000"
```

Reply:
I don't really see much benefit in creating a variable simply for the sake of making the `system` value more noticeable. I'm also not really in favor of creating variables in `livecheck` blocks (within `strategy` blocks is fine but `livecheck` blocks should just contain DSL method calls).

If we wanted to have a variable, we could do something like `system_version = (MacOS.version >= 14) ? "1441" : "1366"` at the top of the cask definition (like we do for some livecheck values in other casks) and that would allow us to use one `livecheck` block, outside of the `on_*` blocks.

That said, I think we can just try using `system=#{MacOS.full_version.to_s.delete(".")}` for now and see how that holds up over time. If we run into problems, we can always fall back to these other approaches.

Reply:
I don't think we can use the MacOS system variables anymore because CI has structures to run as different architecture? Unless it does account for this.